arbnode/node.go

```diff
@@ -37,13 +37,16 @@ import (
 	"github.com/offchainlabs/nitro/arbutil"
 	"github.com/offchainlabs/nitro/broadcastclient"
 	"github.com/offchainlabs/nitro/broadcaster"
+	"github.com/offchainlabs/nitro/cmd/util"
```

Comment:
I'm not familiar with how `cmd/util` is used elsewhere, but does it make sense to import it outside the cmd folder? We could instead put the DataSignerFunc type definition in `arbutil`.

Reply:
:+1:

Reply:
moved to `nitro/util/signature/`